### DISCUSSION TITLE
Invalid characters in drun - Ubuntu 20.04

### DISCUSSION THREAD

**Discussion by bapt1ste** [davatorium/rofi#1296] [Q&A]
Hello Guys,

Updated from ubuntu 18.04 to 20.04 and since then, rodi drun shows invalid characters for some programs (such as Slack, gvim, or Spotify) after the program name.
Tested with latest rofi 1.6.1 and with blank configuration/cache.
Is there any conf I should change or any cache I should remove ?

Thanks for helping.

**Comment by DaveDavenport** [comment]
Can you make a screenshot? and maybe attach the desktop file of these applications.
I have not noticed this myself upgrading to 20.04

**Reply by bapt1ste** [reply]
Here is a screenshot.
![2021-03-29-140743_1920x1080_scrot](https://user-images.githubusercontent.com/8005279/112834522-617ddf80-9098-11eb-93e3-8d54448b69af.png)
(Using XFCE instead of Gnome)
[slack.desktop.txt](https://github.com/davatorium/rofi/files/6221566/slack.desktop.txt)

Command: ./rofi -modi drun -show drun

**Reply by DaveDavenport** [reply]
interesting.  can you pastebin your config (rofi --help output).

**Reply by bapt1ste** [reply]
[rofi-1.5.4.config.txt](https://github.com/davatorium/rofi/files/6221585/rofi-1.5.4.config.txt)
[rofi-1.6.1.config.txt](https://github.com/davatorium/rofi/files/6221586/rofi-1.6.1.config.txt)

Tested with those two configs

**Reply by DaveDavenport** [reply]
could it possibly be a font issue?  can you try it with `rofi -no-config -show drun`?

**Reply by bapt1ste** [reply]
Same issue . 
![2021-03-29-142510_1920x1080_scrot](https://user-images.githubusercontent.com/8005279/112836285-a9056b00-909a-11eb-905e-3b3d718ee5e1.png)

**Reply by bapt1ste** [reply]
Intersting, switching themes, found that it is a font issue, switching to "OpenSans 11" removes the problem.

**Reply by DaveDavenport** [reply]
So default font resolving mono 12, does not have an italic set.
